gatsby/createPages.js

```diff
@@ -26,12 +25,15 @@ module.exports = async ({ actions, graphql }) => {
   const { entries } = result.data.craft;
 
   entries.forEach(entry => {
-    const { uri } = entry;
+    const { uri, slug, __typename } = entry;
 
-    createPage({
-      context: { uri },
-      path: uri,
-      component: path.resolve('src/templates/blog-post.js'),
-    });
+    // Craft CMS section with slug "pages" and URI-Format "{parent.uri}/{slug}"
+    if (__typename == 'Craft_Pages') {
+      createPage({
+        context: { slug },
```

Comment:
ich denke dir uri brauchst du hier auch noch im ctx... oder? um in dem template die query machen zu können